app/src/components/ArtworkInfo.tsx

```diff
@@ -19,7 +19,7 @@ class ArtworkInfo extends React.Component<ArtworkInfoProps, {}> {
       <Grid container direction="row">
         <ListItemAvatar>
           {/* TODO: replace with thumbnail image?? */}
-          <Avatar alt={this.props.artwork.title}>{this.props.artwork.edition}</Avatar>
+          <Avatar alt={this.props.artwork.title}>{this.props.artwork.artworkCreationDate}</Avatar>
```

Comment:
I can't comment on the line because It wasn't changed but if you look at line #34 you still reference `artwork.artist_name` even though it is changed to `artwork.artistName`

Reply:
You also reference `artworkCreationDate` even though it's actually named `createdDate`